source/_posts/2016-09-15-reinventing-our-continuous-learning-practice.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: Reinventing our continuous learning practice
+comments: true
+author:
+  name: Sebastián Martínez
+  email: sebastian@wyeworks.com
+  twitter_handle: smartinez87
+  github_handle:  smartinez87
+  image:  /images/team/sebastian-martinez.jpg
+  description: VP of Engineering at WyeWorks. Ruby on Rails hacker. ExceptionNotification maintainer. Coffee & bacon lover.
+published: true
+---
+
+A bit over a year ago I wrote [this article](https://wyeworks.com/blog/2015/7/16/technical-thursdays-or-how-we-do-continuous-learning/) about continuous learning and
+talked about our Technical Thursdays. If you haven't read it yet, I
+encourage you to do so! In there I talked about the importance of having
+a great team that is curious and passionate about learning.
+
+## Not everything was perfect though
+Some time ago we started to notice a decrease in
+the attendance, people started to skip some sessions. That made us
+question what happened: Did people care less about learning new things?
+Luckly, we were able to see beyond that and realised there were some
+problems with the approach.
+First of all, it was at a fixed time, every
+other Thursday at 9:30am. Let's face it, not everyone *loves* to get
+up that early. It is also known now that people have different times of
```

Comment:
What do you think about "It is also known now that people feel more productive at different times in the day?"